### ISSUE TITLE
Check For Updates isn't working

### ISSUE THREAD

**Issue by mikicho** [HaxeFoundation/haxedevelop.org#14]
Hi,
I saw [this](https://groups.google.com/forum/#!topic/haxelang/_FNsBCm78-A), so I'm trying to install it with "Check For Updates" option in help->Check For Updates. and it's say there is no new version of haxedevelop. my current version is 5.1.1.1(through help->about)

**Comment by markknol** [comment]
@neverbirth any idea why? This file is uptodate http://haxedevelop.org/latest.txt

**Comment by Neverbirth** [comment]
Yeah, we already realized about this problem and Mika fixed it several days ago. Sadly it's a bug in both FD 5.1.1 and 5.2.0 codebase, FD 5.2.1 should be out soon fixing this and a few other improvements.
